api/app.py

```diff
@@ -72,6 +72,10 @@ def get_delegates_representees_mandates(delegate_id):
         error_config = app.config['SETTINGS']['errors']['legal_person_format_validation_failed']
 
         validate_person_company_code(delegate_id, error_config)
+        person_rows = get_person(db, delegate_identifier=delegate_id)
```

Comment:
In my opinion this check is not needed and should be removed.

If get_mandates below doesn't find anything then it will return empty list anyways.

Reply:
Agree. I thought that making decision based on view might lead to incorrect results but actually view uses more strict SELECT, so it it reasonable to use results from this view.